src/core/queryClient.ts

```diff
@@ -535,10 +599,8 @@ export class QueryClient {
   getQueryDefaults(
     queryKey?: QueryKey
   ): QueryObserverOptions<any, any, any, any, any> | undefined {
-    return queryKey
-      ? this.queryDefaults.find(x => partialMatchKey(queryKey, x.queryKey))
-          ?.defaultOptions
-      : undefined
+    const queryDefaults = this.findFirstMatchingQueryDefaults(queryKey)
+    return queryDefaults
```

Comment:
I don't see the need for an extra function here, as this would basically be:

```
return this.findFirstMatchingQueryDefaults(queryKey)
```

and we don't intend on using this function again. So I'd suggest we inline it please.

Reply:
There is no "need". It is just the way I'm used to coding with the "1 function - 1 purpose" mantra.
I'll inline both functions, no pb.